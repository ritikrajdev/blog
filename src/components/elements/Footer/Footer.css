.footer {
  background-color: #121212;
  padding-top: 2rem;
  padding-bottom: 2rem;
  color: white;
}
