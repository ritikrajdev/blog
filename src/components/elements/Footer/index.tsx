import React from 'react';

import './Footer.css';

export default function Footer() {
  return <div className='body-padding footer'>&copy; artifact.com 2019</div>;
}
