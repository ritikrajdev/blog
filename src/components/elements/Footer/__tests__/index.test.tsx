import { render } from '@testing-library/react';
import React from 'react';
import Footer from '..';

describe('Footer', () => {
  it('should render correctly', () => {
    const screen = render(<Footer />);
    expect(screen.asFragment()).toMatchSnapshot();
  });
});
