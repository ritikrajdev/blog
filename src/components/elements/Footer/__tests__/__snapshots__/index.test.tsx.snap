// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="body-padding footer"
  >
    Â© artifact.com 2019
  </div>
</DocumentFragment>
`;
